.travis.yml

```diff
@@ -58,6 +58,8 @@ jobs:
               - sleep 3
           script:
               - npm run test:e2e -- $SINCE_ARG
+          after_script:
+              - if [[ "$TRAVIS_TEST_RESULT" -gt 0 ]]; then echo "Not works"; else echo "Works"; fi
```

Comment:
Can we add more information why its not working and which tests are failing ? Or is this been already covered with as a result of `npm run test:e2e`

Reply:
We were thinking of using this for when we saved the image to an external location. But now it's really unused, maybe can I remove it for now?

Reply:
Removed!